pep-0012.rst

```diff
@@ -568,35 +568,43 @@ the numbers will always match.
 Images
 ------
 
-If your PEP contains a diagram, you may include it in the processed
-output using the "image" directive::
+If your PEP contains a diagram or other graphic, you may include it in the
+processed output using the ``image`` directive:
+
+.. code-block:: rst
 
     .. image:: diagram.png
 
-Any browser-friendly graphics format is possible: .png, .jpeg, .gif,
-.tiff, etc.
+Any browser-friendly graphics format is possible; PNG should be
+preferred for graphics, JPEG for photos and GIF for animations.
+Currently, SVG must be avoided due to compatibility issues with the
+PEP build system.
```

Comment:
Out of scope here, so just a note as accessibility is mentioned in the next paragraph, SVGs can be [good for accessibility](https://css-tricks.com/accessible-svgs/).

(Although it might require more work to create an accessible SVG than is usually put into images here.)

Reply:
Mmm, good point. One more reason for PEP 676, which should get them working, hopefully.